set.mm

```diff
@@ -76850,7 +76850,7 @@ as a separate axiom in an axiom system (such as Kunen's) that uses this
     ex focdmex syl6ci imp ) ABDEZBCFZAGFZUDUEDHZGFZUGADIZJZUFUDUEUHUDUGBKUEUHUD
     ABDABDLMUGBCNOTUDUGAUIPUJABDQUGAUIRSUGAGUIUAUBUC $.
 
-  $( The range of a 1-1 onto function is a set iff its domain is a set.
+  $( The codomain of a 1-1 onto function is a set iff its domain is a set.
```

Comment:
Because the function is onto, "range" isn't wrong, but for reasons I'm having trouble articulating, "codomain" doesn't rub me the wrong way as much as in `f1ocnvb`.